portability-core/src/main/java/org/dataportabilityproject/serviceProviders/flickr/FlickrPhotoService.java

```diff
@@ -85,13 +85,23 @@
     RequestContext.getRequestContext().setAuth(auth);
   }
 
-  private static int getPage(Optional<PaginationInformation> paginationInformation) {
+  FlickrPhotoService(Flickr flickr, PhotosetsInterface photosetsInterface,
```

Comment:
This can be marked @VisibleForTesting or at least comment that it is for testing only

Can we have the original contructor call this one to minimize duplicate code.

Reply:
Done.